### DISCUSSION TITLE
LocalAI do not support GPU

### DISCUSSION THREAD

**Discussion by kewaike** [mudler/LocalAI#1209] [Q&A]
First of all, thank you very much for this great product.

I used docker-compose ，Build with CUDA，The compilation process is normal，but when LocalAI loads models ，Issue happend : 
CUDA：false

<img width="526" alt="image" src="https://github.com/go-skynet/LocalAI/assets/8566143/ca5b3ef8-6fb3-4e8d-9e5c-6e8b117f8a72">

<img width="1377" alt="image" src="https://github.com/go-skynet/LocalAI/assets/8566143/d2c94a2a-5ef6-4203-860c-0bb40874e22e">

Nvidia Driver and tookit are installed correctly，
.env file：
<img width="1195" alt="image" src="https://github.com/go-skynet/LocalAI/assets/8566143/c707c21f-b684-4e69-a456-fa0453113dcf">

docker-compose.yaml file：
<img width="450" alt="image" src="https://github.com/go-skynet/LocalAI/assets/8566143/e2b41961-1f32-443c-a9c5-9206a6e695f2">

in container ：
<img width="641" alt="image" src="https://github.com/go-skynet/LocalAI/assets/8566143/df0f0a9c-ef39-41f5-8d85-934e53b7e16c">

docker-compose.yaml file：
<img width="432" alt="image" src="https://github.com/go-skynet/LocalAI/assets/8566143/4d7087d5-aad6-40bf-b199-ddbd8bf68e6d">

System Information：
OS：CentOS7.9

I also installed stablediffusion （build，not docker），it works well with gpu .

 thanks

[a comment by an automated account is not shown]

**Comment by Aisuko** [comment]
Thanks for your feedback. We have been created an issue for you. https://github.com/go-skynet/LocalAI/issues/1218. And I will close this discussion.

**Comment by saket424** [comment]
@kewaike @Aisuko Any resolution on this issue?
